### PR TITLE
Make account validity renewal emails work when email notifs are disabled

### PR DESCRIPTION
**Pull request by babolivier**
This is a quick fix to the email config only being loaded when email notifs are enabled.

A better long-term solution would be to properly separate email config from notifs and account validity configs.

### CODE REVIEW

File: synapse/config/emailconfig.py [hunk 1]
```diff
@@ -29,19 +31,57 @@
...
+            "require_transport_security", False
+        )
+        if "app_name" in email_config:
+            self.email_app_name = email_config["app_name"]
+        else:
+            self.email_app_name = "Matrix"
+
+        self.email_notif_from = email_config.get("notif_from", None)
+        # make sure it's valid
+        parsed = email.utils.parseaddr(self.email_notif_from)
```
Comment (erikjohnston) [inline synapse/config/emailconfig.py]: Surely we only do this if `email_notif_from is not None`?

File: synapse/config/emailconfig.py [hunk 2]
```diff
@@ -29,19 +31,57 @@
...
+
+        self.email_template_dir = os.path.abspath(template_dir)
+
         self.email_enable_notifs = email_config.get("enable_notifs", False)
+        account_validity_renewal_enabled = config.get(
+            "account_validity", {},
+        ).get("renew_at")
 
-        if self.email_enable_notifs:
+        if self.email_enable_notifs or account_validity_renewal_enabled:
```
Comment (erikjohnston) [inline synapse/config/emailconfig.py]: Do we use `notif_template_*` if `account_validity_renewal_enabled` is enabled?

File: synapse/config/emailconfig.py (hunk 1 above)
Comment (babolivier) [reply synapse/config/emailconfig.py]: Yeah that looks like a cleaner way to do it.

File: synapse/config/emailconfig.py (hunk 2 above)
Comment (erikjohnston) [reply synapse/config/emailconfig.py]: Oh, sorry, I didn't see that you had inserted another if clause. woops

**Pull request merged**